spec/models/card_model_spec.rb

```diff
@@ -0,0 +1,16 @@
+# encoding: utf-8
+require "rails_helper"
+
+RSpec.describe Card, type: :model do
+  context "#check_translation method" do
+    let(:card) {card = Card.new(original_text: "USA", translated_text: "Америка") }
```

Comment:
Useless assignment to variable - `card`.<br>Line is too long. [83/80]<br>Space missing inside {.